core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -306,7 +302,7 @@ trait WhiskActionsApi
      * - 200 [] or [WhiskAction as JSON]
      * - 500 Internal Server Error
      */
-    override def list(namespace: EntityPath, excludePrivate: Boolean)(implicit transid: TransactionId) = {
+    override def list(user: Identity, namespace: EntityPath, excludePrivate: Boolean)(implicit transid: TransactionId) = {
```

Comment:
Why and where is this additional parameter `user` needed? 
Is it to unify these methods?